datasets/yahoo_answers_topics/README.md

```diff
@@ -0,0 +1,140 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- en
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets:
+- extended|other-yahoo-answers-corpus
+task_categories:
+- text-classification
+task_ids:
+- topic-classification
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [Add homepage URL here if available (unless it's a GitHub repository)]()
+- **Repository:** [If the dataset is hosted on github or has a github homepage, add URL here]()
```

Comment:
```suggestion
- **Repository:** https://github.com/LC-John/Yahoo-Answers-Topic-Classification-Dataset
```

Reply:
(just thought I'd add this at least since this dataset doesn't have a good canonical home page iirc)